Add Rscript.exe to environment variables (#2)
EDUCE-UBC/EDUCE-UBC.github.io: intermediate_R_ws_setup.Rmd

---
title: "Workshop setup instructions"
subtitle: "Workshop: Intermediate R Programming"
output: html_document
---

- Make sure that you have installed the [latest versions of R and RStudio](r_and_rstudio.html). 
- In RStudio, install the [devtools and educer packages](https://educe-ubc.github.io/educer.html). This will install all of the tidyverse packages and other resources.
- Install the lmerTest and roxygen2 packages by entering the following code into the console:
    
  ```{r install-lmer-roxygen, eval = FALSE}
  if (!require(lmerTest)) install.packages("lmerTest")
  if (!require(roxygen2)) install.packages("roxygen2")
  ```

> - Download the data file we will use during the workshop by clicking on the following button and save it in your Desktop or another convenient location.

Reviewer: ```suggestion
- Click on the button below to download the data file we will use during the workshop and save it on your Desktop or another convenient location.
```
